src/app/modules/main/chat_section/chat_content/users/module.nim

```diff
@@ -45,27 +50,55 @@ method isLoaded*(self: Module): bool =
 
 method viewDidLoad*(self: Module) =
   # add me as the first user to the list
+  var admin = false
```

Comment:
@alaibe all changes done here in `viewDidLoad` may be easily reverted and changed with the changes I did [here](https://github.com/status-im/status-desktop/pull/4649/files) in `getMembersPublicKeys` and on top of that because of added `isAdmin` and `joined` props for `item`, you just need to update `initItem` in `viewDidLoad`. So less code and easier for understanding without a lot of "if".

Reply:
as u prefer